e2e/ci-e2e/vitest.e2e.config.ts

```diff
@@ -1,22 +1,17 @@
 /// <reference types="vitest" />
-import { defineConfig } from 'vite';
-import { tsconfigPathAliases } from '../../tools/vitest-tsconfig-path-aliases.js';
+import { createE2eConfig } from '../../testing/test-setup-config/src/lib/vitest-setup-presets.js';
 
-export default defineConfig({
-  cacheDir: '../../node_modules/.vite/ci-e2e',
-  test: {
-    reporters: ['basic'],
-    testTimeout: 60_000,
-    globals: true,
-    alias: tsconfigPathAliases(),
-    pool: 'threads',
-    poolOptions: { threads: { singleThread: true } },
-    cache: {
-      dir: '../../node_modules/.vitest',
+export default createE2eConfig(
+  'ci-e2e',
+  {
+    projectRoot: new URL('../../', import.meta.url),
+    cacheKey: 'ci-e2e',
+  },
+  {
+    test: {
+      testTimeout: 60_000,
```

Comment:
this could be a default for the e2e target

Reply:
Hmmm.... I would put 20 as default. Or if there are more projects with 40 lets go with 40.
Did not realise this is already an overwrite..

Reply:
I checked that and:

| Timeout | Occurences |
----------|-------------
| 20s | 6 |
| 40s | 1 |
| 60s | 1 |
| 80s | 2 |
| 120s | 1 |

I suggest then use 20s as default (20_000ms)